src/Rector/Rector/MethodCall/TableRegistryLocatorRector.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Cake\Upgrade\Rector\Rector\MethodCall;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\StaticCall;
+use Rector\Rector\AbstractRector;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\ConfiguredCodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+final class TableRegistryLocatorRector extends AbstractRector
+{
+    public function getRuleDefinition(): RuleDefinition
+    {
+        return new RuleDefinition('Refactor `TableRegistry::get()` to `TableRegistry::getTableLocator()->get()`', [
+            new ConfiguredCodeSample(
+                <<<'CODE_SAMPLE'
+TableRegistry::get('something');
+CODE_SAMPLE
+                ,
+                <<<'CODE_SAMPLE'
+TableRegistry::getTableLocator()->get('something');
+CODE_SAMPLE
+            )
+        ]);
+    }
+
+    public function getNodeTypes(): array
+    {
+        return [StaticCall::class];
+    }
+
+    public function refactor(Node $node): ?Node
+    {
+        if(! $node instanceof StaticCall) {
```

Comment:
Should this be run through the phpcs autofixer?

Reply:
I guess we should remove the exclude for the src/Rector folder from the phpcs.xml